src/BenchmarksApps/Kestrel/PlatformBenchmarks/BenchmarkApplication.Json.cs

```diff
@@ -30,7 +29,8 @@ private static void Json(ref BufferWriter<WriterAdapter> writer, IBufferWriter<b
             utf8JsonWriter.Reset(bodyWriter);
 
             // Body
-            JsonSerializer.Serialize<JsonMessage>(utf8JsonWriter, new JsonMessage { message = "Hello, World!" }, SerializerOptions);
+            JsonSerializer.Serialize(utf8JsonWriter, new JsonMessage { message = "Hello, World!" }, SerializerContext.JsonMessage);
+            utf8JsonWriter.Flush();
```

Comment:
The serializer should flush the writer here (and elsewhere), not the user. Will make a change in dotnet/runtime then update this PR.

```suggestion
```